### PR TITLE
[#7] [UI] As a user, I can see the home header

### PR DESCRIPTION
**Pull request by Wadeewee**
Close https://github.com/hoangnguyen92dn/survey-flutter-ic/issues/7

## What happened 👀

- Display the white status bar
- Display the Date labels at the top left
- Display the profile picture at the top right

## Insight 📝

- Create a `HomeHeader` widget.
- Add `neuzeit_bold.otf`.

The status bar is not white when building the UI on the `ios`, So to fix it. We need to set `AnnotatedRegion<SystemUiOverlayStyle>` with `SystemUiOverlayStyle.light`. 

📌 Please note the current way it will doesn't work if we have an `AppBar`, but we don't have to use it from the `design`, So it's good to use it.

For more refs, please check 👇 

- https://stackoverflow.com/a/62122222.
- https://blog.devgenius.io/different-ways-to-change-the-status-bar-and-navigation-bar-color-ios-and-android-in-flutter-a786e098f573

## Proof Of Work 📹

| Screen  | Before | After |
| ------------- | ------------- | ------------- |
|  `SignInScreen` | Simulator Screen Shot - iPhone 12 Pro Max - 2023-03-30 at 21 27 54  | Simulator Screen Shot - iPhone 12 Pro Max - 2023-03-30 at 21 37 39 |
| `HomeScreen` | Simulator Screen Shot - iPhone 12 Pro Max - 2023-03-30 at 21 28 35  | Simulator Screen Shot - iPhone 12 Pro Max - 2023-03-30 at 21 38 04 |

### CODE REVIEW

File: lib/ui/home/home_screen.dart [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:survey_flutter_ic/ui/home/home_header.dart';
+
+class HomeScreen extends StatelessWidget {
+  const HomeScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return const AnnotatedRegion<SystemUiOverlayStyle>(
```
Comment (Wadeewee) [inline lib/ui/home/home_screen.dart:10]: TODO. Handle `AnnotatedRegion` in `MyApp` instead

Comment (hoangnguyen92dn) [reply lib/ui/home/home_screen.dart:10]: @Wadeewee Will you update this?

File: integration_test/my_home_page_test.dart [hunk 2]
```diff
@@ -1,20 +1,17 @@
...
 import 'package:integration_test/integration_test.dart';
+import 'package:survey_flutter_ic/ui/home/home_screen.dart';
 
 import 'utils/test_util.dart';
 
 void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
-  testWidgets('My Home Page widget', (WidgetTester tester) async {
+  testWidgets('Home Page', (WidgetTester tester) async {
```
Comment (hoangnguyen92dn) [inline integration_test/my_home_page_test.dart:10]: Please rename the test file as well. `my_home_page_test` -> `home_screen_test`

**Review by hoangnguyen92dn — COMMENTED** [review]
The rest LGTM

File: lib/ui/home/home_screen.dart (hunk 1 above)
Comment (Wadeewee) [reply lib/ui/home/home_screen.dart:10]: Yep @hoangnguyen92dn Please allow me update my the end of the day 🙇‍♀️

**Comment by hoangnguyen92dn** [comment]
@Wadeewee Please rebase to resolve conflict 😄

**Review by hoangnguyen92dn — APPROVED** [review]
LGTM 🚀

**Pull request merged**